### PR TITLE
Refactor(import) change commonjs to esm

### PR DESCRIPTION
**Pull request by Zapotoczny-Rafal**
It has changed a bit and I can't automatically merge it, but I hope I helped.

### CODE REVIEW

File: bin/minify.js [hunk 1]
```diff
@@ -59,12 +59,12 @@ async function minify() {
...
         return log.error(optionsError.message);
     
-    uglifyFiles(files, options);
+    await uglifyFiles(files, options);
 }
 
 async function processStream(chunks) {
-    const minify = require('..');
-    const tryToCatch = require('try-to-catch');
+    await import('..');
```
Comment (coderaiser) [inline bin/minify.js:66]: are you sure that it works?

File: bin/minify.js [hunk 2]
```diff
@@ -93,8 +93,8 @@ function logAll(array) {
         log(item);
 }
 
-function help() {
-    const bin = require('../help');
+async function help() {
+    const bin = await import('../help.json');
```
Comment (coderaiser) [inline bin/minify.js:97]: Unfortunately this is not valid `js`

**Comment by coderaiser** [comment]
Could you please merge master into your brunch, or just start from `master` and use [putout to convert ESM to Commonjs](https://github.com/coderaiser/putout#converting-commonjs-to-esm). This tool will be helpful [simport](https://github.com/coderaiser/simport);

**Comment by Zapotoczny-Rafal** [comment]
I am leaving it because now, unfortunately, I won't have time for it

**Pull request closed without merging**